tokio/tests/time_pause.rs

```diff
@@ -215,6 +215,40 @@ async fn interval() {
     assert_pending!(poll_next(&mut i));
 }
 
+#[tokio::test]
+async fn test_time_advance_sub_ms() {
+    time::pause();
```

Comment:
```suggestion
#[tokio::test(start_paused = true)]
async fn test_time_advance_sub_ms() {
```
Small cleanliness nit